frontend/views/components/Modal/ModalBaseTemplate.vue

```diff
@@ -1,7 +1,8 @@
 <template lang='pug'>
-  transition(name='zoom' appear @after-leave='$destroy()')
```

Comment:
Why shouldn't we destroy the modal when it is removed?

Reply:
Because it's done already with the :is=null